templates/tapestry.css

```diff
@@ -382,6 +381,10 @@ input[type="checkbox"][disabled] {
     bottom: 8px;
 }
 
+.tooltip-wrapper {
```

Comment:
Sorry can you actually move this up in the code since it's the parent element of some of the above elements? (I believe?)